qiskit/compiler/assembler.py

```diff
@@ -191,12 +265,16 @@ def assemble(
         )
         end_time = time()
         _log_assembly_time(start_time, end_time)
-        return assemble_circuits(
-            circuits=bound_experiments,
-            qobj_id=qobj_id,
-            qobj_header=qobj_header,
-            run_config=run_config,
-        )
+        with warnings.catch_warnings():
+            # The Qobj is deprecated
+            # warnings.filterwarnings("ignore", category=DeprecationWarning, module="qiskit")
```

Comment:
Any specific reason why the `module` field is not being used here? In any case we should remove the comment:

```suggestion
```

Reply:
hmm.. it is actually not needed. Removed in 0fe2e02f7caf279f72eb92f64f0f5279d65320a7